packages/react-card/src/components/Card/renderCard.tsx

```diff
@@ -1,17 +1,15 @@
 import * as React from 'react';
 import { getSlotsCompat } from '@fluentui/react-utilities';
-import { cardShorthandProps } from './useCard';
 import type { CardState } from './Card.types';
 
 /**
  * Render the final JSX of Card
  */
 export const renderCard = (state: CardState) => {
-  const { slots, slotProps } = getSlotsCompat(state, cardShorthandProps);
+  const { slots, slotProps } = getSlotsCompat(state);
```

Comment:
since this is the first phase of a new component, please follow the new slot guidelines from #19483 It's a beta high priority task to migrate components to the new API

Reply:
Compat was used on purpose as the PR was still pending approval back then. If this is not breaking, I would rather leave it for when the implementation of the rest of the functionality starts to avoid delaying it (given that it's not one of the Beta components and we're not closing Card with this).
Could this be postponed to the next PR?

Reply:
I would say that when there's less functionality this change is easier to make (having had to do this for the full Menu package). But you are driving the work on card, so I'll leave it to your timeline 👍

Reply:
Plan would be to pick that up right after the PR gets merged and before we start with additional functionality, so impact should be the same.